### ISSUE TITLE
conflicting messages on ballot manifest upload

### ISSUE THREAD

**Issue by sfsinger19103** [FreeAndFair/ColoradoRLA#553]
Working in `master`, last commit `2b17ddf`, got three conflicting messages on ballot manifest upload. Screenshot below. 

After upload attempt, uploaded_file table in database indicates hash_status "MISMATCH".

![upload_msgs](https://user-images.githubusercontent.com/14353822/29856092-7e6fd5d4-8d05-11e7-8ee2-f6796581805f.png)

**Comment by sfsinger19103** [comment]
I see that I tried to upload the hash file (not the ballot manifest file) so no wonder the upload failed. But the messages are still confusing -- why the green success message?

**Comment by dmzimmerman** [comment]
That is confusing. Certainly, the client can _upload_ any file it wants... it's the parsing step that would cause the problem. So in this case, the upload succeeded, but it did not parse properly. Perhaps that first success message just muddies the water.

**Comment by ranweiler** [comment]
I think there might be some confusion here. @sfsinger19103, that hash is the hash for the file `Ballot Manifest.csv`. it is the _contents_ of the file `Ballot Manifest Hash.txt`, which is what you are trying to upload. Try uploading `Ballot Manifest.csv` instead.

**Comment by ranweiler** [comment]
In other words: the hash you've entered really doesn't match the file you are trying to upload. That is the first check which occurs; file well-formedness is only checked when parsing is attempted, which happens after upload.

The _upload_ step means literally that: we uploaded the file. _Import_ is the more important thing. Maybe we shouldn't give an alert for upload unless that fails. The idea, though, is that for very long running CSV parses, we want to let you know "hey, we got the file, we're working on it", and then when we're done, say "hey, your file looks good, we parsed and imported it". Improving the UX/user feedback around that already has an issue.
